### ISSUE TITLE
SimpleDNSCrypt v0.4.7 stops "Charging [0]" after opening It

### ISSUE THREAD

**Issue by gnavascues** [bitbeans/SimpleDnsCrypt#219]
I have a problem after updating from SimpleDNSCrypt v0.4.6 to v 0.4.7.

   After starting SimpleDNSCrypt a "Charging [0]" window appears. But SimpleDNSCrypt never start.

   I have tried to uninstall v0.4.7 SimpleDNSCrypt and then I have tried with SimpleDNScrypt v0.4.6 and v0.4.5 and same problem. Is It possible that the problem is with SimpleDNSCrypt v0.4.7 installer?

**Comment by bitbeans** [comment]
@gnavascues can you give me some more details?
- OS
- Language
- maybe a screenshot

**Comment by dani-garcia** [comment]
I have the same problem, using version 0.4.7 in Windows 10 in Spanish. It gets stuck in the loading screen like this:
![captura](https://user-images.githubusercontent.com/725423/36079149-3256b93a-0f80-11e8-9e0a-1eb3d8c0bbf2.PNG)

I was using 0.4.3 with no problems, before updating to the latest version.

Looking through the installation files, i found a logs folder and these are the contents of the file with the current date, if it helps:
```
2018-02-11 23:04:25.5590 LoaderViewModel System.FormatException: La cadena de entrada no tiene el formato correcto.
   en System.Text.StringBuilder.FormatError()
   en System.Text.StringBuilder.AppendFormatHelper(IFormatProvider provider, String format, ParamsArray args)
   en System.String.FormatHelper(IFormatProvider provider, String format, ParamsArray args)
   en System.String.Format(String format, Object arg0)
   en SimpleDnsCrypt.ViewModels.LoaderViewModel.<InitializeApplication>d__7.MoveNext()
```

**Comment by gnavascues** [comment]
OS: Windows 10 64 bit
   Language: Spanish
   Screenshot: Same as "dani-garcia" screenshot

**Comment by bitbeans** [comment]
@gnavascues @dani-garcia there was an translation error in the spanish language file. i fixed this. i will publish a new version as soon as possible. 

Workaround: 
1. Open: C:\Users\<user>\AppData\Local\SimpleDnsCrypt\<randomstring>\version\user.config
2. Change to:

```
<?xml version="1.0" encoding="utf-8"?>
<configuration>
    <userSettings>
        <SimpleDnsCrypt.Properties.Settings>
            <setting name="PreferredLanguage" serializeAs="String">
                <value>en</value>
            </setting>
        </SimpleDnsCrypt.Properties.Settings>
    </userSettings>
</configuration>
```
3. save and start SimpleDnsCrypt again.

**Comment by gnavascues** [comment]
OK. Thank you. After your Workaround SimpleDNSCrypt works well.

   Problem solved

**Comment by bitbeans** [comment]
@gnavascues thank you for reporting. spanish will be fixed in 0.4.8
